lms/templates/navigation-edx.html

```diff
@@ -142,7 +142,9 @@ <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
 <![endif]-->
 % endif
 
-%if not user.is_authenticated():
+## The forgot password modal JavaScript assumes that JQuery is loaded,
+## which is not necessarily the case when using RequireJS.
+%if not enable_require_js and not user.is_authenticated():
   <%include file="forgot_password_modal.html" />
```

Comment:
As far as I can tell, the "forgot password" modal in this snippet is used only on the Dashboard.  I strongly suspect this is leftover code that doesn't need to be included on every page, but I'm nervous about taking it out in case I'm misunderstanding the LMS template structure.